static/js/compose.js

```diff
@@ -574,7 +574,7 @@ exports.validate = function () {
 };
 
 exports.initialize = function () {
-    $('#stream,#subject,#private_message_recipient').bind({
+    $('#stream,#subject,#private_message_recipient').on({
```

Comment:
I think the one-line-of-code-per-mapping is simpler, and I'm not sure we use the function map idiom anywhere else in our code.